### PR TITLE
Release/4.3.5

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#70645](https://central.tri.be/issues/70645)

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -3271,7 +3270,11 @@ public function VenueMetaBox() {
 				}
 
 				foreach ( $this->venueTags as $tag ) {
+<<<<<<< HEAD
```
Comment (barryhughes) [inline src/Tribe/Main.php]: Unresolved conflict (see also code beginning line 3322)

File: tests/integration/Tribe/Events/Aggregator/Record/Queue_CleanerTest.php [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+		$this->assertTrue( $failed );
+		$this->assertEquals( Records::$status->failed, ( get_post( $record->post->ID ) )->post_status );
+		$this->assertEmpty( get_post_meta( $record->post->ID, '_tribe_aggregator_queue', true ) );
+		$this->assertEmpty( get_post_meta( $record->post->ID, '_transient_tribe_aggregator_queue', true ) );
+		$this->assertEmpty( get_post_meta( $record->post->ID, '_transient_timeout_tribe_aggregator_queue', true ) );
+	}
+
+	    /**
+	         * maybe_fail_stalled_record will empty queue meta for failed records
```
Comment (barryhughes) [inline tests/integration/Tribe/Events/Aggregator/Record/Queue_CleanerTest.php]: This probably doesn't result from the merge itself - but the indentation is all out of whack here

**Comment by bordoni** [comment]
Ready for the next 👁‍🗨

**Comment by borkweb** [comment]
:+1:

**Pull request merged**